loadgen/loadgen.cc

```diff
@@ -586,13 +587,31 @@ void PerformanceSummary::ProcessLatencies() {
     lp.sample_latency = pr.sample_latencies[sample_count * lp.percentile];
   }
 
+  query_count = pr.queries_issued;
+
+  // Count the number of overlatency queries for the Server or SingleStream
```

Comment:
I don't get why SingleStream scenario would need to track overlatency queries. In SingleStream, there is no target latency (well, there is only "expected latency" which tells LoadGen how many queries to generate so that the runtime is roughly `min_duration`, but this latency is not a requirement or something like that). Therefore, I think we only need to report overlatency queries for Server scenario. 

@psyhtest for visibility

Reply:
Or to put it more clear, in SingleStream/MultiStream, it is fine if the SUT has many queries whose latencies are above the expected latency while the result is still VALID. In contrast, in Server scenario, the SUT can only have up to 1% of overlatency queries or the result will become INVALID.

Reply:
Mmmm you’re right, I misunderstood the early stopping [algorithm](https://docs.google.com/document/d/1noN3SVGy8MRF958T6GOV2aTN1K1mZrXC_leDwRXCo74/edit?usp=sharing&resourcekey=0-9t1VQNUdSlYIqxFeV-HvRQ) for the SingleStream/MultiStream scenario and I thought we needed to track the overlatency queries (it isn’t). I will revert the changes